api/swagger/server/restapi/certHandler.go

```diff
@@ -0,0 +1,226 @@
+package restapi
```

Comment:
This code should not be here. You should put it in:
- /int (for /internal) directory if it's not ready to be an external package yet
- /pkg if you think that the code can be reused as is by another project.